crypto/ui/ui_lib.c

```diff
@@ -427,9 +427,13 @@ static int print_error(const char *str, size_t len, UI *ui)
 int UI_process(UI *ui)
 {
     int i, ok = 0;
+    const char *state = NULL;
```

Comment:
this should probably be something generic like "processing" to future-proof from future bunglers.